src/components/container/MyDataHelpsAssistant/MyDataHelpsAssistant.css

```diff
@@ -0,0 +1,190 @@
+:root {
+    --ce-grey-1: #FBFCFC;
+    --ce-grey-2: #f8f9fb;
+    --ce-grey-3: #F4F6F7;
+    --ce-grey-4: #F1F2F5;
+    --ce-grey-5: #EDEFF2;
+    --ce-grey-10: #DFE1E7;
+    --ce-grey-20: #C6CAD5;
+    --ce-grey-30: #A9ADBF;
+    --ce-grey-40: #8E90A6;
+    --ce-grey-50: #73748B;
+    --ce-grey-55: #6B6B81;
+    --ce-grey-60: #595A6B;
+    --ce-grey-70: #434351;
+    --ce-grey-80: #2E2D35;
+    --ce-grey-90: #1D1C21;
+    --mdh-blue-1: #fafcff;
+    --mdh-blue-2: #f4f8ff;
+    --mdh-blue-3: #f1f6ff;
+    --mdh-blue-4: #ebf3fe;
+    --mdh-blue-5: #e5effe;
+    --mdh-blue-10: #d1e2fd;
+    --mdh-blue-20: #cde3ff;
+    --mdh-blue-30: #7daffa;
+    --mdh-blue-40: #4c91f8;
+    --mdh-blue-50: #146ef6;
+    --mdh-blue-55: #0c64eb;
+    --mdh-blue-60: #0954c8;
+    --mdh-blue-70: #063e98;
+    --mdh-blue-80: #032a6a;
+    --mdh-blue-90: #001a48;
+    --mdh-blue-95: #001843;
+    --mdh-blue-96: #00173f;
+    --mdh-blue-97: #001438;
+    --mdh-blue-98: #001232;
+    --mdh-blue-99: #00102d;
+    --border: 1px solid var(--ce-grey-10);
+    --border-radius: 10px;
+    --touch: 44px;
+    --lg-padding: 20px;
+}
+
+.mdh-assistant-collapsed {
+    position: fixed;
+    bottom: 20px;
+    right: 20px;
+    width: var(--touch);
+    height: var(--touch);
+    background: var(--mdhui-color-primary);
+    display: flex;
+    box-shadow: var(--mdhui-box-shadow-1);
+    border-radius: 200px;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    cursor: pointer;
+}
+
+.mdh-assistant {
+    position: fixed;
+    bottom: 0;
+    background-color: white;
+    font-size: 14px;
+    width: calc(100% - 1rem);
+    border: var(--border);
+    border-radius: 12px;
+    overflow: hidden;
+    margin: .5rem;
+    z-index: 3;
+    max-width: 800px;
+    right: 0;
+    box-shadow: var(--mdhui-box-shadow-1);
+}
+
+.mdh-assistant-header {
+    display: flex;
+    align-items: center;
+    height: 44px;
+    padding: 0 12px;
+    justify-content: space-between;
+    border-bottom: var(--border);
+    font-weight: 600;
+    color: var(--ce-grey-55);
+}
+
+.mdh-assistant-header .collapse-button {
+    color: var(--mdhui-color-primary);
+    cursor: pointer;
+}
+
+.mdh-assistant-header svg {
+    padding-right: 0.5rem
+}
+
+.mdh-assistant #log {
+    overflow-y: auto;
+    overflow-x: hidden;
+    height: 80vh;
+}
+
+/* typography */
+.mdh-assistant p,
+.mdh-assistant ul {
+    margin: .25rem 0;
+    word-break: break-word;
+    line-height: 1.4;
+    font-size: 15px;
+    color: var(--ce-grey-80);
+}
+
+/* form elements */
+.mdh-assistant input,
+.mdh-assistant button {
+    min-height: var(--touch);
+    min-width: var(--touch);
+    border: 0;
+    border-radius: var(--border-radius);
+}
+
+.mdh-assistant input {
```

Comment:
There is another .mdh-assistant input CSS selector below this should be combined with